eng/native/configureplatform.cmake

```diff
@@ -229,6 +243,11 @@ if(CLR_CMAKE_TARGET_OS STREQUAL Darwin)
     set(CLR_CMAKE_TARGET_DARWIN 1)
 endif(CLR_CMAKE_TARGET_OS STREQUAL Darwin)
 
+if(CLR_CMAKE_TARGET_OS STREQUAL iOS)
+    set(CLR_CMAKE_TARGET_UNIX 1)
+    set(CLR_CMAKE_TARGET_IOS 1)
+endif(CLR_CMAKE_TARGET_OS STREQUAL iOS)
```

Comment:
I'd expect iOS to set `CLR_CMAKE_TARGET_DARWIN` too. Not sure whether it would make the rest of the diff with `if` conditions smaller or larger though.

Reply:
Yes please, at least on cmake side, we are consistent with `Darwin` naming. It's only the code and RIDs where the (obsolete) `osx` name is used. 😁

Reply:
This was a conscious decision because `CLR_CMAKE_TARGET_DARWIN` really means macOS in the rest of the build scripts. We could rename it to `CLR_CMAKE_TARGET_OSX` in a separate PR.

Reply:
I'll do the renaming in a follow-up PR.